test/InterchainTokenServiceUpgradeFlow.js

```diff
@@ -86,14 +85,7 @@ describe('Interchain Token Service Upgrade Flow', () => {
         );
 
         axelarServiceGovernance = await axelarServiceGovernanceFactory
-            .deploy(
-                gateway.address,
-                governanceChain,
-                governanceAddress,
-                buffer,
-                signers.map((signer) => signer.address),
-                threshold,
-            )
+            .deploy(gateway.address, governanceChain, governanceAddress, minimumTimeDelay, operator.address)
```

Comment:
Why did the number of parameters being passed to `deploy` change - was `thresehold` not needed anymore?